README.md

```diff
@@ -71,471 +42,22 @@ for an introduction to our philosophy of pipeline data quality testing.
-				Orchestrator
-			</td>
-		</tr>
-		<tr>
-			<td style="text-align: center; height: 40px; background-color: #fff;">
-				<img height="40" src="./docs/readme_assets/prefect-logo-gradient-navy.jpg" />
-			</td>
-			<td style="width: 200px;">
-				<a href="https://docs.greatexpectations.io/docs/oss/deployment_patterns/how_to_use_great_expectations_with_prefect/">
-					Prefect
-				</a>
-			</td>
-			<td>
-				Orchestrator
-			</td>
-		</tr>
-		<tr>
-			<td style="text-align: center; height: 40px; background-color: #fff;">
-				<img height="40" src="./docs/readme_assets/zenml.jpg" />
-			</td>
-			<td style="width: 200px;">
-				<a href="https://docs.greatexpectations.io/docs/oss/integrations/integration_zenml/">
-					ZenML
-				</a>
-			</td>
-			<td>
-				Orchestrator
-			</td>
-		</tr>
-		<tr>
-			<td style="text-align: center; height: 40px; background-color: #fff;">
-				<img height="40" src="./docs/readme_assets/slack.jpg" />
-			</td>
-			<td style="width: 200px;">
-				<a href="https://docs.greatexpectations.io/docs/oss/guides/validation/validation_actions/how_to_trigger_slack_notifications_as_a_validation_action/">
-					Slack
-				</a>
-			</td>
-			<td>
-				Plugin
-			</td>
-		</tr>
-	</tbody>
-</table>
-
-
-What is GX _not_?
--------------------------------------------------------------
-
-Great Expectations is _not_ a pipeline execution framework. Instead, it integrates seamlessly with DAG execution tools
-like [Spark]( https://spark.apache.org/), [Airflow](https://airflow.apache.org/), [dbt]( https://www.getdbt.com/)
-, [prefect](https://www.prefect.io/), [dagster]( https://github.com/dagster-io/dagster)
-, [Kedro](https://github.com/quantumblacklabs/kedro), [Flyte](https://flyte.org/), etc. GX carries out your data quality
-pipeline testing while these tools execute the pipelines.
-
-Great Expectations is _not_ a database or storage software. It processes your data in place, on your existing systems.
-Expectations and Validation Results that GX produces are metadata about your data.
-
-Great Expectations is _not_ a data versioning tool. If you want to bring your data itself under version control, check
-out tools like [DVC](https://dvc.org/), [Quilt](https://github.com/quiltdata/quilt),
-and [lakeFS](https://github.com/treeverse/lakeFS/).
-
-Great Expectations is _not_ a language-agnostic platform. Instead, it follows the philosophy of “take the compute to the
-data” by using the popular Python language to support native execution of Expectations in pandas, SQL (via SQLAlchemy),
-and Spark environments.
-
-Great Expectations is _not_ exclusive to Python programming environments. It can be invoked from the command line
-without a Python environment. However, if you’re working into another ecosystem, you may want to explore
-ecosystem-specific alternatives such as [assertR](https://github.com/ropensci/assertr) (for R environments)
-or [TFDV](https://www.tensorflow.org/tfx/guide/tfdv) (for Tensorflow environments).
-
+## Get started
```

Comment:
@joshzzheng, we don't have a succinct discussion about OSS architecture similar to what we provide for GX Cloud. Also, a README is not the correct location for architecture information. If you feel strongly about this, I'll see if I can draft something - it could take some time.

Reply:
Ok nvm, no worries. We can skip.